### ISSUE TITLE
Making all commands run in background causes the output to be irregular

### ISSUE THREAD

**Issue by rahulr56** [dylanaraps/neofetch#782]
## Description

Making all commands run in background causes the output to be irregular.

#### Neofetch version
3.2.0

## Screenshot
![screenshot from 2017-07-21 17-28-44](https://user-images.githubusercontent.com/8971145/28458074-f21190e8-6e3a-11e7-8ee8-60dd4d645f8d.png)

## Config file
[Config file](https://github.com/rahulr56/ConfigFiles/blob/master/neoFetchConfig)

## Verbose log
1. Run `neofetch -vv 2> neofetchlog`
2. Upload the contents of `neofetchlog` to pastebin, gist or equivalent.

[neofetchlog.txt](https://github.com/dylanaraps/neofetch/files/1165074/neofetchlog.txt)

**Comment by dylanaraps** [comment]
For context @rahulr56's config file is using this: https://github.com/dylanaraps/neofetch/wiki/Customizing-Info#speed-up-the-script-by-running-the-functions-asynchronously

**Comment by dylanaraps** [comment]
I'm not sure if this broke recently or if it's been broken for a while. Either way I think that we should just remove this from the wiki and mark it as `wontfix`.

**Comment by konimex** [comment]
Try to add a single `wait` at the bottom of the function.

**Comment by dylanaraps** [comment]
Good catch, that looks like the issue.

**Comment by rahulr56** [comment]
@konimex the whole point is to make it faster. Adding a wait would result in the same time as not having that option only.

**Comment by dylanaraps** [comment]
The wait is needed since Neofetch needs to do things directly after **all**
of the info is detected. This is why you’re getting the output in your
screenshot.

Even with the ‘wait’ this will be faster than default since the functions
are being run in parallel as opposed to sequentially.

**Comment by rahulr56** [comment]
@dylanaraps , I agree! Forgot that point totally!
